index.html

```diff
@@ -4,18 +4,48 @@
     <meta charset="UTF-8" />
     <link rel="icon" href="public/icon-list.png" alt="list-icon" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="stylesheet" href="src/css/style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital@1&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="src/css/style.css" />
     <title>Abdullahi Todo List App</title>
   </head>
 
   <body class="bg-blue-900 text-lg md:mx-auto md:w-[600px]">
     <main role="main">
+      <section
+        class="fixed left-0 right-0 top-0 z-50 flex h-full items-center justify-center bg-blue-101 not-italic"
+        id="authentication-page"
+      >
+        <div class="relative flex items-center justify-center">
+          <div class="fixed bottom-[335px] w-[340px] rounded-lg bg-blue-0 p-[22px] shadow">
```

Comment:
Let's not fix this. Instead an margin should be used to center it. Also, the sizes should use percentage instead.

Reply:
Done